meilisearch-http/tests/settings/get_settings.rs

```diff
@@ -16,10 +16,11 @@ async fn get_settings() {
     let (response, code) = index.settings().await;
     assert_eq!(code, 200);
     let settings = response.as_object().unwrap();
-    assert_eq!(settings.keys().len(), 5);
+    assert_eq!(settings.keys().len(), 6);
     assert_eq!(settings["displayedAttributes"], json!(["*"]));
     assert_eq!(settings["searchableAttributes"], json!(["*"]));
     assert_eq!(settings["attributesForFaceting"], json!({}));
+    assert_eq!(settings["distinctAttribute"], serde_json::Value::Null);
```

Comment:
```suggestion
    assert_eq!(settings["distinctAttribute"], json!(null));
```

Since we are already using the json! macro everywhere I guess it's more logical to use it as well

Reply:
Sure, seems legit, I will update